Examples/DataRepresentation/Image/Image4.cxx

```diff
@@ -347,8 +346,8 @@ int main(int, char *[])
 
   // Software Guide : BeginCodeSnippet
   const ImageType::IndexType LeftEyeIndex = GetIndexFromMouseClick();
-  ImageType::PointType LeftEyePoint;
-  image->TransformIndexToPhysicalPoint(LeftEyeIndex,LeftEyePoint);
+  ImageType::PointType       LeftEyePoint;
```

Comment:
I very much dislike alignment somewhere in the middle of a line of code. It would mean that whenever line 348 is modified, line 349 would also be modified to keep the alignment in sync. Please no!

Reply:
@N-Dekker, Personally I agree, but this has been discussed several times before, and the historical community agreement is that the benefits in readability outweigh the pain.

Reply:
FWIW, I agree with @N-Dekker. It will mess up blame & history to re-indent like that.

Reply:
I agree, it's very hard to maintain this alignment. Even if this was discussed before, since formatting will change now anyway, it would be a good time to re-check what most people prefer.

There are many similar questions where people can have different preferences, so maybe all the controversial decisions could be put up on a public vote. We could give different weight to votes depending on involvement in ITK project (e.g., a core ITK developer could have 10x more vote than an ITK user).

Reply:
Yes, this is annoying inside functions/methods. Unfortunately, clang-format does not distinguish that from declarations inside classes.

@hjmjohnson Maybe we could completely abolish alignment of declarations because of this?

Reply:
@dzenanz @lassoan @N-Dekker @thewtex I want to be VERY CLEAR:  I am doing my best to provide better automation of style, and not dictate the style.  I spent a lot of time finding the set of options that minimizes changes to the current style while providing AUTOMATION.

I'm willing to entertain other global style changes, but to the extent possible I would rather not let the scope of this PR require that every style be re-considered.

Please vote now, and recruit others to support/reject this coding change.

NOTE:  Making this change will also require changing KWSTYLE rules, as the current suggestions will cause the KWSTYLE checks to fail.

--- NOW my personal opinion:  Let's change KWSTYLE and remove it's enforcement, and change .clang-format `AlignConsecutiveDeclarations: true` to `AlignConsecutiveDeclarations: false`

Reply:
Thank you for taking the time for this, I look forward to not thinking about style anymore. 👔 🕴 

Are we going to need KWStyle any more?

My issue with looking through history, is not a single change like this one ( seems to be a very good plan for it ). It is more the frequent white space changes cause by this case and the one below. Thank you for being open to a couple be for the big commit.